mobile/src/components/Navigator.tsx

```diff
@@ -13,14 +13,15 @@ const Navigator = () => {
   const { isLoggedIn } = useContext(AuthContext);
 
   return (
-    <Stack.Navigator initialRouteName="Login" screenOptions={{ headerShown: false }}>
+    <Stack.Navigator initialRouteName="Chess" screenOptions={{ headerShown: false }}>
       {isLoggedIn ? (
         <>
           <Stack.Screen name="HomeScreen" component={HomeScreen} />
           <Stack.Screen name="Chess" component={Chess} />
         </>
       ) : (
         <>
+          <Stack.Screen name="Chess" component={Chess} />
```

Comment:
We should remove this from the login flow and put the initialRouteName back to Login for the PR.